src/js/common/components/questions/Phone.jsx

```diff
@@ -37,6 +37,7 @@ class Phone extends React.Component {
             autocomplete="tel"
             placeholder="xxxxxxxxxx"
             field={this.props.value}
+            required={this.props.required}
```

Comment:
Did we verify that this doesn't create issues for other projects using this component?